ddcci-multi-monitor@tim-we/files/ddcci-multi-monitor@tim-we/applet.js

```diff
@@ -111,7 +117,9 @@ class DDCMultiMonitor extends Applet.IconApplet {
         super(orientation, panelHeight, instance_id);
         this.detecting = false;
         this.set_applet_icon_symbolic_name("display-brightness");
-        this.set_applet_tooltip("Adjust monitor brightness via DDC/CI");
+        this.set_applet_tooltip(DEFAULT_TOOLTIP);
+        this.actor.connect('scroll-event', (...args) => this._onScrollEvent(...args));
+        this.lastTooltipTimeoutID = setTimeout(() => {}, 1);
```

Comment:
You can initialize this with `undefined` or `null`, passing an invalid timeout ID to `clearTimeout` [does nothing; no exception is thrown](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout#notes).

Reply:
Oh interesting, I was expecting it to be like a promise in that way.